### DISCUSSION TITLE
开发者新人入手路径

### DISCUSSION THREAD

**Discussion by fanux** [labring/sealos#2865] [Announcements]
开发 sealos 有一个我比较推荐的路径：

1. 编译源代码 ： [编译教程](https://github.com/labring/sealos/blob/main/DEVELOPGUIDE.md)
2. 用你编译过的二进制运行 quick start : https://github.com/labring/sealos#quickstart
3. 用 quick start 安装好的 kubernetes 集群学习和使用 kubernetes, 了解基本使用/基本操作，核心组件和基本原理即可 [基础知识](https://kubernetes.io/zh-cn/docs/tutorials/kubernetes-basics/)  这里花两天左右快速入门，实操很重要。 如果不知道什么是容器的补充一点 docker 的知识，半天，知道镜像和容器的概念，实际操作过 build push pull run 就差不多了。
4. 熟悉贡献代码的流程：[贡献流程](https://github.com/labring/sealos/blob/main/CONTRIBUTING.md)
5. 了解 sealos 整体是干啥的 [bilibili](https://www.bilibili.com/video/BV1SR4y1D7Yh/?spm_id_from=333.337.search-card.all.click)

以上基本就是入门了。

**Comment by fanux** [comment]
https://sealos.io/zh-Hans/docs/cloud/Intro  sealos cloud 的整体介绍。

**Reply by fanux** [reply]
可以加入微信群获取帮助：

<img width="458" alt="image" src="https://user-images.githubusercontent.com/8912557/227723448-0fdcdf60-526e-4837-9ffb-d7f45cf07a82.png">

**Comment by bxy4543** [comment]
在对整个项目有了较为详细的了解之后，可以优先考虑从[“good first issue”](https://github.com/labring/sealos/issues?q=is%3Aopen+is%3Aissue+label%3A%22good+first+issue%22)入手。这些[“good first issue”](https://github.com/labring/sealos/issues?q=is%3Aopen+is%3Aissue+label%3A%22good+first+issue%22)是为了方便初次贡献者为该项目做出贡献而特别标注的。

完成了“good first issue”，可以继续探索项目的[其他问题](https://github.com/labring/sealos/issues?q=is%3Aopen+is%3Aissue+label%3Akind%2Fbug)并为其做出贡献。这将有助于你更深入地了解该项目的内部工作原理和贡献流程。如果你在贡献过程中遇到了任何问题或困难，可以向项目的社区或维护者求助，他们通常会乐于提供帮助和指导。
